e2e/tests/content-manager/uniqueness.spec.ts

```diff
@@ -76,15 +73,15 @@ test.describe('Uniqueness', () => {
        * Modify the value and try again, this should save successfully
        * Either take the new value provided in the field object or generate a random new one
        */
-      // TODO find a better way to generate random values for all field types
+      // TODO: find a better way to generate random values for all field types
       const newValue =
-        field?.newValue ||
-        (isNaN(Number(field.value))
+        'newValue' in field
+          ? field.newValue
+          : isNaN(Number(field.value))
           ? String.fromCharCode(Math.floor(Math.random() * 26) + 97) + field.value.substring(1)
-          : `${Number(field.value) + 10}`);
+          : `${Number(field.value) + 10}`;
```

Comment:
Oh man. To each their own and up you, personally I prefer a function that returns for each condition over nested ternaries or if-else-if

```ts
function getNewValue(field) {
	if ('newValue' in field) {
		return field.newValue
	}

	if (isNaN(Number(field.value))) {
	 return String.fromCharCode(Math.floor(Math.random() * 26) + 97) + field.value.substring(1)
	}

	return `${Number(field.value) + 10}`
}
```

Reply:
I wrote the uniqueness e2e spec so I can handle this, I did have it in mind to rework this 🙏🏻

Reply:
I didn't write this explicitly, but you're right, lets clean it up 🧽